### DISCUSSION TITLE
Move between `q-inputs` or `custom inputs` inside `q-table` using keyboards arrows

### DISCUSSION THREAD

**Discussion by OrestTsolkovskyi** [quasarframework/quasar#15525] [General - Components / Directives / etc]
Hi guys. I'm stuck with an interesting problem.

I want to be able to move between inputs inside a q-table using keyboard arrows, not `Tab`.
I'm using Vue3 Composition Api <script setup> + TS.

I would be very grateful for your help.

**Comment by pdanpdan** [comment]
make a codepen please
but I don't think it would be possible except if you only have inputs on a single column
how would you choose left/right cursor movement inside input and between inputs?

**Reply by OrestTsolkovskyi** [reply]
I'm not sure how to chose it this is my codepen. 
https://codepen.io/orest-tsolkovskyi/pen/gOdxpJb

**Reply by metalsadman** [reply]
your codepen doesn't work.
